Sources/MintKit/Mint.swift

```diff
@@ -113,7 +113,12 @@ public class Mint {
                 .filter { !$0.lastComponent.hasPrefix(".") }
                 .sorted()
                 .map { $0.lastComponent }
-            let packageName = String(packagePath.lastComponent.split(separator: "_").last!)
+            let packageName: String = {
+                var packageInfo = packagePath.lastComponent.split(separator: "_")
+                let packageName = packageInfo.popLast()!
+                let authorName = packageInfo.last.flatMap { $0 + "/" } ?? ""
+                return String(authorName + packageName)
```

Comment:
This works well for github repos, but what about for other sources like non github sites or ssh only urls? For those cases, it would still look for the second last path right? That should be ok. What do you think?